app/main.py

```
@@ -182,10 +209,25 @@ async def lifespan(_app: FastAPI):
             "Background task scheduler disabled (ENABLE_BACKGROUND_TASKS=false)"
         )
 
+    # Start event-driven lead dispatcher
+    try:
+        await initialize_lead_dispatcher(
+            process_fn=process_single_lead, max_workers=BACKLOG_WORKER_COUNT
+        )
+        logger.info(f"Lead dispatcher started with {BACKLOG_WORKER_COUNT} workers")
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)